gui/src/installer/mod.rs

```diff
@@ -138,7 +138,16 @@ impl Installer {
                 data_dir.push(self.context.bitcoin_config.network.to_string());
                 // In case of failure during install, block the thread to
                 // deleted the data_dir/network directory in order to start clean again.
-                std::fs::remove_dir_all(data_dir).expect("Correctly deleted");
+                log::warn!("Install failed ! Trying now to clean the corrupted data directory");
+                if let Err(e) = std::fs::remove_dir_all(data_dir.clone()) {
```

Comment:
nit: Isn't the clone unnecessary?
```suggestion
                if let Err(e) = std::fs::remove_dir_all(&data_dir) {
```